tests/integration/test_fim/test_registry/test_registry_basic_usage/test_basic_usage_registry_duplicated_entries.py

```diff
@@ -0,0 +1,99 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import os
+
+import pytest
+from wazuh_testing import global_parameters
+import wazuh_testing.fim as fim
+from wazuh_testing.tools.configuration import load_wazuh_configurations
+from wazuh_testing.tools.monitoring import FileMonitor
+
+# Marks
+
+pytestmark = [pytest.mark.win32, pytest.mark.tier(level=0)]
+
+# Variables
+
+key = "HKEY_LOCAL_MACHINE"
+sub_key_1 = "SOFTWARE\\Classes\\testkey"
+sub_key_2 = "SOFTWARE\\Classes\\Testkey"
+
+test_regs = [os.path.join(key, sub_key_1), os.path.join(key, sub_key_2)]
```

Comment:
This variable is only used to unpack it later in line 27. I think is easier to just create the variables `reg1` and `reg2` directly:
```suggestion
reg1, reg2 = os.path.join(key, sub_key_1), os.path.join(key, sub_key_2)
```
Also, please, give them an easier name to read.